examples/api/lib/cupertino/list_section/list_section_base.0.dart

```diff
@@ -0,0 +1,95 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Can you add tests for your samples? They help us ensure we don't have broken sample code in our documentation.

You can find examples of these tests in this directory, which is where you'll want to put your tests as well: https://github.com/flutter/flutter/tree/451ccced68e4d7f45204e516271e6f349a6ba381/examples/api/test

Reply:
But of course, will try to add them tomorrow morning.

Reply:
Thank you! 🙏

Reply:
Unfortunately, I was not able to add tests this morning so I will have to postpone this till evening due to urgent errands.

Reply:
I have added the tests - hopefully this is what you were looking for.

Reply:
Awesome! Thank you!